app/src/Home/Home.js

```diff
@@ -1,20 +1,221 @@
+import { useState } from "react";
+import { SiOnlyfans, SiInstagram, SiX, SiLinkedin, SiGithub } from "react-icons/si";
 import Footer from "../Components/Footer/Footer";
 import Navbar from "../Components/Navbar/Navbar";
 
+const projectList = [
+    ["Skibidi", "Nisi dolor fugiat cillum id eu labore ex esse quis esse in. Consequat magna aute est dolore cillum sunt."],
+    ["Toilet", "Duis mollit in irure ea irure qui adipisicing eiusmod adipisicing. In sint incididunt nulla commodo in voluptate nostrud eiusmod ipsum qui laborum esse consequat labore."],
+    ["Dop Dop", "Incididunt id ea consectetur ea eiusmod nulla dolore elit reprehenderit consequat duis veniam. Qui sint officia consectetur pariatur excepteur est ullamco consequat dolore irure."],
+    ["Yes Yes", "Dolor exercitation occaecat velit tempor ea commodo enim do occaecat proident magna. Eu aliqua fugiat exercitation aliqua proident do aliquip velit."]
+]
+
+const serviceList = [
+    "Software Development",
+    "Web Development",
+    "Application Production",
+    "Data Analytics",
+    "Technical Consulting",
+    "Project Management"
+]
+
```

Comment:
Move to a different file, and query from those. No hard-coded strings for items that may change with transalation.